### DISCUSSION TITLE
My Teacher verification always is rejected.

### DISCUSSION THREAD

**Discussion by newlulu** [community/community#51717] [GitHub Education]
I have tried to apply for the GitHub Education Teacher Pack many times. But the applications all are rejected. I have received rejected emails which show that my GitHub user profile with full name as it appears in my academic affiliation document plus a short bio. 

My full name in the academic affiliation document is my Chinese name.  I have tried to update my GitHub user profile with my Chinese name. But it still has been rejected. 

Can anybody explain the rejected reason clearly?

**Comment by samus-aran** [comment]
Hi @newlulu 

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Hope this helps! Let us know how you get along 👍

**Reply by t-hedeon** [reply]
Thank you!)

**Comment by tomoyanonymous** [comment]
Hi, @samus-aran . I am probably in the similar situation (my proof document is in Japanese). Let me know clearer on the translation.

> Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

If my original document is in non-English, Which language should I set for GitHub user name and the name of my academic institution, Japanese or English? Is it OK that I include both of the language in the same space, or shouldn't?

Best.
